recommended.js

```diff
@@ -0,0 +1,17 @@
+const eslintConfigPrettier = require('eslint-config-prettier');
+const eslintPluginPrettier = require('./eslint-plugin-prettier');
+
+// Merge the contents of eslint-config-prettier into every
+module.exports = {
+  ...eslintConfigPrettier,
+  plugins: {
+    ...eslintConfigPrettier?.plugins,
```

Comment:
```suggestion
    ...eslintConfigPrettier.plugins,
```

I don't think it could be `null` or `undefined` at all? And it hides hidden bugs from upstream.

Reply:
Yep you're right, for some reason I was rushing and hung up on the idea that `plugins` isn't defined  on eslintConfigPrettier.